doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -459,9 +395,6 @@ Thus, implementations may raise errors when spatial index limits are exceeded.
 The recommended HTTP status code for those situations is ``413``, *Request entity too large*, and
 the reported error on the response payload must be ``NoResourcesAvailable``.
 
-*Note: At the time of writing it is still under discussion whether supporting both syntaxes is
-really needed* 
-
```

Comment:
Release Candidate text shouldn't include explicit references to open dicussions. That references to open discussions are fine, but within the context of the WIP version of the speficiaiton (i.e. the one that will remain in "/latest").